cms/djangoapps/contentstore/views/tests/test_transcripts.py

```diff
@@ -766,3 +766,77 @@ def test_fail_for_non_video_module(self):
         resp = self.client.get(link, {'data': json.dumps(data)})
         self.assertEqual(resp.status_code, 400)
         self.assertEqual(json.loads(resp.content).get('status'), 'Transcripts are supported only for "video" modules.')
+
+
+class TestSaveTranscripts(BaseTranscripts):
+    """
+    Tests for '/transcripts/save' url.
+    """
+    def save_subs_to_store(self, subs, subs_id):
+        """
+        Save transcripts into `StaticContent`.
+        """
+        filedata = json.dumps(subs, indent=2)
+        mime_type = 'application/json'
+        filename = 'subs_{0}.srt.sjson'.format(subs_id)
+
+        content_location = StaticContent.compute_location(self.course.id, filename)
+        content = StaticContent(content_location, filename, mime_type, filedata)
+        contentstore().save(content)
+        del_cached_content(content_location)
+        return content_location
```

Comment:
Let's move this method into BaseTranscripts and remove it from other test classes.